docs/variants/protectli_vp66xx/releases.md

```diff
@@ -12,7 +12,44 @@ For details about our release process please read
 </center>
 
 Test results for this platform can be found
-[here](https://docs.google.com/spreadsheets/d/1wI0qBSLdaluayYsm_lIa9iJ9LnPnCOZ9eNOyrKSc-j4/edit#gid=1316498194).
+[here](https://docs.google.com/spreadsheets/
+d/1wI0qBSLdaluayYsm_lIa9iJ9LnPnCOZ9eNOyrKSc-j4/edit#gid=1316498194).
```

Comment:
Now it looks confusing, we have a google sheet and a repo with results? We should have only one source of test results I think

Reply:
We are slowly transfering to using the repo, I think

Reply:
Well, if the google sheet is going to be replaced, I don't see a reason to keep it here.

Reply:
https://github.com/Dasharo/docs/compare/d34f429e0c882a38b8e7517a5ef96b1a624ccc2f..f7f8c669943fb3890306bca06f76e5429e56fd6a#diff-659c34c80386e22060961792d547a0b7720de5d79b8a127963a4acdec8d39bfbL15-R15